main.c

```diff
@@ -178,6 +179,7 @@ static struct {
 	.cfg.col = 80,
 	.cfg.row = 24,
 	.cfg.scrollback = 0,
+	.cfg.reset_scrollback = true,
```

Comment:
The default should be off.
The default should always be the more minimal behaviour.